docs/how-to/testing-verification/test-kernel-in-proposed.md

```diff
@@ -61,10 +61,13 @@ Then install the kernel as per usual. If the kernel version in -proposed is the
 highest in any pocket, install it by running:
```

Comment:
Hi @nwager , i propose to change the structure a little in this section

1. It would help to briefly indicate why you would choose to install the metapackage vs ABI-named image
2. For your original content, I think if you install the kernel via ABI-named image, the "kernel version being the highest in the proposed pocket" is not relevant I think since you specify the actual version.

This is my suggested edit; please do review & adjust as needed.

````
First, update the sources cache:

```{code-block} none
sudo apt update
```

Then proceed to install the kernel using either a metapackage or a specific
ABI-named image.

### Install via kernel metapackage

Use this approach if you want to receives automatic updates for the latest
version of the kernel in that series.

If the kernel version in -proposed is the highest in any pocket, run:

```{code-block} none
sudo apt install linux-<flavour>
```

If you want a specific (earlier) version of a metapackage, include the version
in the command:

```{code-block} none
sudo apt install linux-<flavour>=<version>
```

### Install via ABI-named kernel image

Use this method to install a specific kernel version without being tied to the
kernel series metapackage.

```{code-block} none
sudo apt install linux-image-<version>
```
````

Thanks.

Reply:
Good point, it makes more sense when things aren't haphazardly lumped together. Added this change pretty much verbatim in the recent commit e5b6767af7f43a97ff3ca538d866dd186536cdf6